organization.go

```diff
@@ -1,25 +1,117 @@
 package zd
 
+import (
+	"fmt"
+)
+
+type OrganizationWrapper struct {
+	Organization *Organization `json:"organization"`
+}
+
 type OrganizationResponse struct {
-	Organizations []Organization `json:"organizations,omitempty"`
-	NextPage      *string        `json:"next_page,omitempty"`
-	PreviousPage  *string        `json:"previous_page,omitempty"`
-	Count         *int           `json:"count,omitempty"`
+	Organization *Organization `json:"organization"`
+}
+
+type OrganizationListResponse struct {
+	Organizations []Organization `json:"organizations"`
+	NextPage      string        `json:"next_page,omitempty"`
+	PreviousPage  string        `json:"previous_page,omitempty"`
+	Count         int           `json:"count,omitempty"`
 }
 
 type Organization struct {
-	URL                *string           `json:"url,omitempty"`
-	ID                 *int              `json:"id,omitempty"`
-	Name               *string           `json:"name,omitempty"`
-	SharedTickets      *bool             `json:"shared_tickets,omitempty"`
-	SharedComments     *bool             `json:"shared_comments,omitempty"`
-	ExternalID         *string           `json:"external_id,omitempty"`
-	CreatedAt          *time.Time        `json:"created_at,omitempty"`
-	UpdatedAt          *time.Time        `json:"updated_at,omitempty"`
+	URL                string           `json:"url,omitempty"`
+	ID                 int              `json:"id,omitempty"`
+	Name               string           `json:"name,omitempty"`
+	SharedTickets      bool             `json:"shared_tickets,omitempty"`
+	SharedComments     bool             `json:"shared_comments,omitempty"`
+	ExternalID         string           `json:"external_id,omitempty"`
+	CreatedAt          string           `json:"created_at,omitempty"`
+	UpdatedAt          string           `json:"updated_at,omitempty"`
 	DomainNames        []string          `json:"domain_names,omitempty"`
-	Details            *string           `json:"details,omitempty"`
-	Notes              *string           `json:"notes,omitempty"`
-	GroupID            *string           `json:"group_id,omitempty"`
+	Details            string           `json:"details,omitempty"`
+	Notes              string           `json:"notes,omitempty"`
+	GroupID            string           `json:"group_id,omitempty"`
 	Tags               []string          `json:"tags,omitempty"`
 	OrganizationFields map[string]string `json:"organization_fields,omitempty"`
 }
+
+// TicketService struct
+type OrganizationService struct {
+	client *Client
+}
+
+// finds an organization in zendesk byt id
+func (s *OrganizationService) GetOrganizationById(organization_id string) (*Organization, *Response, error) {
```

Comment:
func GetOrganizationById should be GetOrganizationByID<br>don't use underscores in Go names; method parameter organization_id should be organizationID